maas-schemas/schemas/maas-backend/products/provider.json

```diff
@@ -100,9 +100,23 @@
           "required": ["fixedFareAmount", "fixedFareCurrency", "maxRadiusMetres"],
           "additionalProperties": false
         },
-        "prebookingSupported": {
-          "description": "Is pre-booking supported",
-          "type": "boolean"
+        "prebooking": {
+          "type": "object",
+          "properties": {
+            "isSupported": {
+              "description": "Is pre-booking supported",
```

Comment:
Though i would say, not having "prebooking" prop means it is not supported, but being clear about it is ok too